docs/src/rules/no-undefined.md

```diff
@@ -77,6 +80,9 @@ if (typeof foo === "undefined") {
 }
 
 global.undefined = "foo";
+
+// Skips the first parameter
```

Comment:
Same comment here. I'm not sure what this is supposed to mean.

Reply:
Would the original comments (plus spelling fixes) be better?